src/local.rs

```diff
@@ -115,6 +125,38 @@ impl LocalAsset {
         }
     }
 
+    /// Creates a new directory, including all parent directories
+    pub fn create_directory(dest: &str) -> Result<PathBuf> {
+        let dest_path = PathBuf::from(dest);
+        match fs::create_dir_all(&dest_path) {
+            Ok(_) => Ok(dest_path),
+            Err(details) => Err(AxoassetError::LocalAssetWriteNewFailed {
+                dest_path: dest_path.display().to_string(),
+                details,
+            }),
+        }
+    }
+
+    /// Removes a file or directory
+    pub fn remove(dest: &str) -> Result<()> {
```

Comment:
do we want to have a `remove_dir` and `remove_file` instead of and/or in addition to `remove`? deleting is destructive so i think being specific can be useful to avoid unwanted actions happening by accident